basics/getting-started/frequent-questions/operations-faq.md

```diff
@@ -92,6 +92,21 @@ curl -X POST "http://localhost:9000/tenants"
 -H "Content-Type: application/json" 
 -d "{\"tenantRole\":\"BROKER\",\"tenantName\":\"foo\",\"numberOfInstances\":1}"
 ```
+### How to I manually run a Periodic Task
+
+Use the `GET /periodictask/names` API to fetch the names of all the Periodic Tasks running on your Pinot cluster.
+
+```
+curl -X GET "http://localhost:9000/periodictask/names" -H "accept: application/json"
+```
+
+To manually run a named Periodic Task use the `GET /periodictask/run` API
```

Comment:
Can we list all the periodic task names here?

Reply:
I mentioned the API for getting the names in the faq since that is probably a better source than listing the names here which may go out of date.

Reply:
Yes, let's add some examples.

Reply:
Done